samples/app-hello-world/nodejs/appManifest/manifest.json

```diff
@@ -1,80 +1,119 @@
-  "staticTabs": [
-    {
-      "entityId": "com.contoso.helloworld.hellotab",
-      "name": "Hello Tab",
-      "contentUrl": "https://${{BOT_DOMAIN}}/hello",
-      "scopes": [
-        "personal"
+  "elementRelationshipSet": {
+      "oneWayDependencies": [
+          {
+              "element": {
+                  "name": "composeExtensions",
+                  "id": "composeExt_ID",
+                  "commandIds": [
+                      "getRandomText"
+                  ]
+              },
+              "dependsOn": [
+                  {
+                      "name": "bots",
+                      "id": "${{AAD_APP_CLIENT_ID}}"
+                  }
+              ]
+          }
+      ],
+      "mutualDependencies": [
+          [
+              {
+                  "name": "bots",
+                  "id": "${{AAD_APP_CLIENT_ID}}"
+              },
+              {
+                  "name": "staticTabs",
+                  "id": "staticTab_ID"
+              },
+              {
+                  "name": "configurableTabs",
+                  "id": "configurableTab_ID"
+              }
+          ]
       ]
-    }
+  },
+  "staticTabs": [
+      {
+          "entityId": "staticTab_ID",
+          "name": "Hello Tab",
+          "contentUrl": "https://${{BOT_DOMAIN}}/hello",
+          "scopes": [
+              "personal"
+          ]
+      }
   ],
   "configurableTabs": [
-    {
-      "configurationUrl": "https://${{BOT_DOMAIN}}/configure",
-      "canUpdateConfiguration": true,
-      "scopes": [
-        "team"
-      ]
-    }
+      {
+          "id": "configurableTab_ID",
+          "configurationUrl": "https://${{BOT_DOMAIN}}/configure",
+          "canUpdateConfiguration": true,
+          "scopes": [
+              "team",
+              "groupChat"
+          ]
+      }
   ],
   "bots": [
-    {
-      "botId": "${{AAD_APP_CLIENT_ID}}",
-      "needsChannelSelector": false,
-      "isNotificationOnly": false,
-      "scopes": [
-        "team",
-        "personal"
-      ]
-    }
+      {
+          "botId": "${{AAD_APP_CLIENT_ID}}",
+          "needsChannelSelector": false,
+          "isNotificationOnly": false,
+          "scopes": [
+              "team",
+              "personal",
+              "groupChat"
+          ]
+      }
   ],
   "composeExtensions": [
-    {
-      "botId": "${{AAD_APP_CLIENT_ID}}",
-      "commands": [
-        {
-          "id": "getRandomText",
-          "description": "Gets some random text and images that you can insert in messages for fun.",
-          "title": "Get some random text for fun",
-          "initialRun": true,
-          "parameters": [
-            {
-              "name": "cardTitle",
-              "description": "Card title to use",
-              "title": "Card title"
-            }
+      {
+          "id": "composeExt_ID",
+          "botId": "${{AAD_APP_CLIENT_ID}}",
+          "commands": [
+              {
+                  "id": "getRandomText",
```

Comment:
Suggest we use a more meaningful value for the id field, despite not necessarily needing UUID.